File: IronVault/UI/ironvault_mobile/lib/models/korisnik_clanarina.dart
import 'package:ironvault_mobile/models/clanarina.dart';
import 'package:json_annotation/json_annotation.dart';

part 'korisnik_clanarina.g.dart';

@JsonSerializable()
class KorisnikClanarina {
  int? korisnikClanarinaId;
  int? korisnikId;
  int? clanarinaId;
  Clanarina? clanarina;
  DateTime? datumUplate;
  DateTime? datumIsteka;

  KorisnikClanarina({this.korisnikClanarinaId});

  factory KorisnikClanarina.fromJson(Map<String, dynamic> json) =>
      _$KorisnikClanarinaFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$KorisnikClanarinaToJson(this);
}

File: IronVault/IronVault.Model/Models/KorisnikClanarina.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public partial class KorisnikClanarina
    {
        public int KorisnikClanarinaId { get; set; }

        public int KorisnikId { get; set; }

        public int ClanarinaId { get; set; }

        public DateTime DatumUplate { get; set; }

        public DateTime DatumIsteka { get; set; }

        public virtual Clanarina Clanarina { get; set; } = null!;

    }
}
